### PR TITLE
docs: Update command-line.md TOC

### PR DESCRIPTION
**Pull request by basilhendroff**
Add build-info and upgrade subcommands to the summary

### CODE REVIEW

**Review by mholt — APPROVED** [review]
Thank you!

File: src/docs/markdown/command-line.md [hunk 1]
```diff
@@ -62,6 +65,9 @@ The ellipses `...` indicates a continuation, i.e. one or more parameters.
 - **[caddy untrust](#caddy-untrust)**
   Untrusts a certificate from local trust store(s)
 
+- **[caddy unpgrade](#caddy-upgrade)**
```
Comment (mholt) [inline src/docs/markdown/command-line.md]:
```suggestion
- **[caddy upgrade](#caddy-upgrade)**
```

Comment (mohammed90) [inline src/docs/markdown/command-line.md]:
```suggestion
- **[caddy upgrade](#caddy-upgrade)**
```

**Review by mholt — CHANGES_REQUESTED** [review]
Actually pls fix typo :)

**Review by francislavoie — APPROVED** [review]

**Pull request merged**